.github/workflows/e2e-feature-tests.yml

```diff
@@ -116,6 +116,14 @@ jobs:
       run: |
         .github/scripts/e2e_tests_brownfield_use_case.sh
 
+    # the script accepts number of requests for  APIs for CREATE_BUCKET, DELETE_BUCKET, GET_INFO
+    # GRANT_ACCESS and REVOKE_ACCESS in order
+    # Example below we we are testing for 2 CREATE_BUCKET, 1 DELETE_BUCKET,
+    # 1 GET_INFO, 2 GRANT_ACCESS and 2 REVOKE_ACCESS API counts
```

Comment:
```suggestion
    # Example below we are testing for those API counts:
    # - 2 CREATE_BUCKET
    # - 1 DELETE_BUCKET
    # - 1 GET_INFO
    # - 2 GRANT_ACCESS
    # - 2 REVOKE_ACCESS
```

Reply:
changed in https://github.com/scality/cosi-driver/pull/83/commits/8cc730891878aa9154d6d951db8032698b2aa2e5
It is indeed more clear, thanks.

Reply:
github shows there is tab instead of spaces for indentation of those

Reply:
That was new editor issue, thanks.